### PR TITLE
Support for SQL Server, include license model param

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -110,7 +110,7 @@ resource "aws_db_instance" "rds" {
   engine                       = var.db_engine
   engine_version               = var.db_engine_version
   instance_class               = var.db_instance_class
-  name                         = local.db_name
+  name                         = can(regex("sqlserver", var.db_engine)) ? null : local.db_name
```
Comment (mattops) [inline main.tf:113]: `name` needs to be `null` for sqlserver, otherwise terraform/aws gives an error.

File: main.tf [hunk 2]
```diff
@@ -120,7 +120,7 @@ resource "aws_db_instance" "rds" {
   db_subnet_group_name         = var.replicate_source_db != "" ? null : aws_db_subnet_group.db_subnet[0].name
   vpc_security_group_ids       = [aws_security_group.rds-sg.id]
   kms_key_id                   = var.replicate_source_db != "" ? null : aws_kms_key.kms[0].arn
-  multi_az                     = true
+  multi_az                     = can(regex("sqlserver-web|sqlserver-ex", var.db_engine)) ? false : true
```
Comment (mattops) [inline main.tf:123]: web edition and express edition do not support `multi_az`

File: main.tf [hunk 3]
```diff
@@ -132,6 +132,7 @@ resource "aws_db_instance" "rds" {
   skip_final_snapshot          = var.skip_final_snapshot
   deletion_protection          = var.deletion_protection
   backup_window                = var.backup_window
+  license_model                = var.license_model
```
Comment (mattops) [inline main.tf:135]: this is a required param for `sqlserver`

**Comment by razvan-moj** [comment]
Superseded by https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/pull/96, we did use all your hints though, thank you!

**Pull request closed without merging**